src/main/java/org/opensearch/ad/AnomalyDetectorExtension.java

```diff
@@ -3,18 +3,23 @@
 import static java.util.Collections.unmodifiableList;
 
 import java.io.IOException;
+import java.util.Collection;
 import java.util.List;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
 import org.opensearch.ad.rest.RestCreateDetectorAction;
 import org.opensearch.ad.settings.AnomalyDetectorSettings;
 import org.opensearch.ad.settings.EnabledSetting;
+import org.opensearch.client.opensearch.OpenSearchClient;
+import org.opensearch.cluster.service.ClusterService;
 import org.opensearch.common.settings.Setting;
+import org.opensearch.sdk.*;
```

Comment:
Given how much confusion I know was generated in creating this PR with similarly-named classes, this star import should be a big red flag. :-)